.github/workflows/on-pr-closed.yaml

```diff
@@ -28,6 +28,10 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 12
     steps:
+    - name: Install oc
+      uses: redhat-actions/oc-installer@v1
+      with:
+        oc_version: '4.6'
```

Comment:
This should always be done after the Checkout step by convention. Do check if this can take a more recent version of OC though, 4.6 is very old already and IIRC the cluster is at 4.14 and planning on moving to 4.16 soon.